src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const population = [...document.getElementsByClassName('population')];
+const totalPopulationClass = document.querySelector('.total-population');
+const averagePopulationClass = document.querySelector('.average-population');
+
+const total = population.reduce((sum, currentPopulation) =>
+  sum + +currentPopulation.innerHTML.split(',').join(''), 0);
+
+const average = total / population.length;
+
+totalPopulationClass.innerHTML = total.toLocaleString('en-EN');
+averagePopulationClass.innerHTML = Math.round(average).toLocaleString('en-EN');
```

Comment:
Math.round() краще перенести, щоб всі операції над змінною були в одному місці в даному випадку!
```suggestion
const average = Math.round(total / population.length);

totalPopulationClass.innerHTML = total.toLocaleString('en-EN');
averagePopulationClass.innerHTML = average.toLocaleString('en-EN');
```